apps/api/src/billing/services/stripe/stripe.service.ts

```diff
@@ -113,29 +119,100 @@ export class StripeService extends Stripe {
       .sort((a, b) => b.created - a.created);
   }
 
+  async getDefaultPaymentMethod(user: PayingUser, ability: AnyAbility): Promise<PaymentMethod | undefined> {
+    const [customer, local] = await Promise.all([
+      this.customers.retrieve(user.stripeCustomerId, {
+        expand: ["invoice_settings.default_payment_method"]
+      }),
+      this.paymentMethodRepository.accessibleBy(ability, "read").findDefaultByUserId(user.id)
+    ]);
+
+    assert(!customer.deleted, 402, "Payment account has been deleted");
+
+    const remote = customer.invoice_settings.default_payment_method as Stripe.PaymentMethod;
+
+    if (remote && local) {
```

Comment:
comment: technically they should always both be there, maybe we should throw an error or a log so we can spot if something is wrong?